R/data_extract_module.R

```diff
@@ -255,10 +255,14 @@ check_data_extract_spec_react <- function(datasets, data_extract) {
 #' Extracting details of the selection(s) in [data_extract_ui] elements.
 #'
 #' @inheritParams shiny::moduleServer
-#' @param datasets (`FilteredData`)\cr
-#'  object containing data, see [teal.slice::FilteredData] for more.
+#' @param datasets (`FilteredData` or `list` of `data.frame`)\cr
```

Comment:
It has to be a list of reactive data.frames
```suggestion
#' @param datasets (`FilteredData` or `list` of `reactive`)\cr
```